### PR TITLE
Styles/ember migration

### LINKED ISSUE #777: Styling fixes in develop-ember
### Issue Description

Here are the list of changes that has to be done:

- Fix card overflow scrollbar styles.
- Header logo link not working issue.
- Add fallback profile image in navbar.
- Remove dev=true flag in dropdown.
- Move more-about info section to feature flag(will remove after migration)

### Expected Behavior

- Fix card overflow scrollbar styles.
- Header logo link should redirect to home page
- Fallback profile image in navbar if profile not present
- Removal of dev=true flag in dropdown.
- more-about info section to be under feature flag(will remove after migration)

### Current Behavior

n/a

### Screenshots

image
> overflow card styling issue

image
- Fallback profile image in navbar issue

### Reproducibility

 - [x] This issue is reproducible
-  [ ] This issue is not reproducible

### Steps to Reproduce
SImply go to https://live.realdevsquad.com to see the issues

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [x] Medium
 - [ ] Low

### Additional Information

[Include any additional context, logs, or details that might be helpful in addressing the issue.]

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

---

Thank you for contributing to our project! We appreciate your help in improving it.

### PR DESCRIPTION
**Pull request by satyam73**
### What is the change?

Here are the list of changes that has been done in this PR:
- Fix card overflow scrollbar styles.
- Header logo link not working issue.
- Added fallback profile image in navbar.
- Removed dev=true flag in dropdown.
- Move more-about info section to feature flag(will remove after migration)

### Is Development Tested?

- [x] Yes
- [ ] No

Before :
Before images can be seen in #777 

After :
image
> Added fallback profile image in navbar

image
> Fixed card overflow scrollbar styles

image
> Removed dev=true flag in dropdown 

image
>  Moved more-about info section to feature flag
**Note :**
What to delete and what not to

- Keep the headings and answer the questions.
- Delete the examples
- Delete the description below headings `What is the change?` & `Before/After Screenshots`

### CODE REVIEW

File: app/constants/urls.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const APP_URLS = {
     TASKS: `${SCHEME}staging-my.${DOMAIN}/tasks`,
     IDENTITY: `${SCHEME}staging-my.${DOMAIN}/identity`,
     MY_STATUS: `${SCHEME}staging-my.${DOMAIN}`,
-    API_BACKEND: `${SCHEME}staging-api.${DOMAIN}`,
+    API_BACKEND: `http://localhost:3000`,
```
Comment (vinit717) [inline app/constants/urls.js:41]: Change this

File: app/styles/navbar.module.css [hunk 2]
```diff
@@ -231,15 +231,15 @@ nav li {
...
-  }
-
   .nav__home__link,
   .active {
     display: block;
   }
 
+  .nav__home {
+    display: none;
+  }
```
Comment (vinit717) [inline app/styles/navbar.module.css:241]: Why display none

File: app/styles/cards.module.css [hunk 3]
```diff
@@ -184,8 +184,17 @@
...
     width: 350px;
+    scrollbar-width: none;
+  }
+
+  .modal::-webkit-scrollbar {
+    display: none;
+  }
+
+  .modal::-webkit-scrollbar-thumb {
+    display: none;
```
Comment (vinit717) [inline app/styles/cards.module.css:197]: Does this works on other browser also

File: app/constants/urls.js (hunk 1 above)
Comment (satyam73) [reply app/constants/urls.js:41]: It's for development environment and I intentionally changed this

File: app/styles/navbar.module.css (hunk 2 above)
Comment (satyam73) [reply app/styles/navbar.module.css:241]: It was added before if you will notice we don't have logo in mobile navbar that's why `display: none`

File: app/styles/cards.module.css (hunk 3 above)
Comment (satyam73) [reply app/styles/cards.module.css:197]: Yes for mozilla scrollbar-width property is set to none

**Review by ajoykumardas12 — APPROVED** [review]

File: app/styles/navbar.module.css (hunk 2 above)
Comment (vinit717) [reply app/styles/navbar.module.css:241]: Here display none is applied to 1024 screen size

**Review by vinit717 — APPROVED** [review]

File: app/styles/navbar.module.css (hunk 2 above)
Comment (satyam73) [reply app/styles/navbar.module.css:241]: yes

**Pull request merged**